modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -133,11 +133,37 @@ const getEditableRoot = function (dom, node) {
   return parent !== root ? editableRoot : root;
 };
 
+const applyAttributes = (editor: Editor, node: DomElement, forcedRootBlockAttrs: Record<string, string>) => {
+  // Merge and apply style attribute
+  Option.from(forcedRootBlockAttrs.style)
+    .map(editor.dom.parseStyle)
+    .each((attrStyles) => {
+      const currentStyles = Css.getAllRaw(Element.fromDom(node));
+      const newStyles = { ...currentStyles, ...attrStyles };
+      editor.dom.setStyles(node, newStyles);
+    });
+
+  // Merge and apply class attribute
+  const attrClasses = Option.from(forcedRootBlockAttrs.class).map((attrC) => attrC.split(/\s+/));
+  const currentClasses = Option.from(node.className).map((currentC) => Arr.filter(currentC.split(/\s+/), (c) => c !== ''));
+  Options.lift2(attrClasses, currentClasses, (attrC, currentC) => {
+    const filteredClasses = Arr.filter(currentC, (c) => !Arr.contains(attrC, c));
```

Comment:
We generally avoid single-character variable names, as they aren't very readable, so maybe change `c` to `clazz` or something similar?